content/en/faqitems/fleet-management-2-scanning-cluster-software-vulns.markdown

```diff
@@ -0,0 +1,7 @@
++++
+question = "How do I scan my cluster for software vulnerabilities?"
+group = "Fleet Management"
```

Comment:
I would change the group to "security" - I think the average user would find this q more readily than under "fleet management"